### PR TITLE
Virtual Application - Output variable fix

### PR DESCRIPTION
**Pull request by vincentdass**

### CODE REVIEW

File: Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts [hunk 1]
```diff
@@ -43,7 +43,7 @@ async function main() {
         await appServiceUtility.pingApplication();
         let kuduService: Kudu = await appServiceUtility.getKuduService();
         kuduServiceUtility = new KuduServiceUtility(kuduService);
-        tl.setVariable('AppServiceApplicationUrl', await appServiceUtility.getApplicationURL());
+        tl.setVariable('AppServiceApplicationUrl', await appServiceUtility.getApplicationURL(!taskParams.isLinuxApp ? taskParams.VirtualApplication : null));
```
Comment (kmkumaran) [inline Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts]: Why this is specific to LinuxApp. What's the behavior for VirtualApp in Windows?

Comment (vincentdass) [reply Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts]: This is specific for Windows app. Virtual application is not available in Linux platform.

File: Tasks/AzureRmWebAppDeployment/operations/AzureAppServiceUtility.ts [hunk 2]
```diff
@@ -57,9 +57,9 @@ export class AzureAppServiceUtility {
         return defer.promise;
     }
 
-    public async getApplicationURL(): Promise<string> {
+    public async getApplicationURL(virtualApplication?: string): Promise<string> {
         let webDeployProfile: any =  await this.getWebDeployPublishingProfile();
-        return await webDeployProfile.destinationAppUrl;
+        return await webDeployProfile.destinationAppUrl + "/" + (virtualApplication ? virtualApplication : "");
```
Comment (Ajay-MS) [inline Tasks/AzureRmWebAppDeployment/operations/AzureAppServiceUtility.ts]: For a virtual application, URL is not terminating with '/' but for a web app, URL is terminating with '/'. Should behaviour be same behavior in both cases?

Comment (vincentdass) [reply Tasks/AzureRmWebAppDeployment/operations/AzureAppServiceUtility.ts]: thanks Ajay. modified the logic.

**Pull request merged**